pkg/app/piped/controller/controller.go

```diff
@@ -341,6 +342,9 @@ func (c *controller) syncPlanners(ctx context.Context) error {
 			)
 			continue
 		}
+
+		controllermetrics.UpdateDeploymentStatus(d.Id, d.Status, d.Kind, d.PlatformProvider)
```

Comment:
Which deployment status do we expect to get here? 🤔

Reply:
status should be pending here.

Reply:
Thanks for pointing out!


> Also, jus to confirm, we don't have states earlier than PENDING, so not PENDING deployment metrics are not available now, right?
Yes. Metrics are available from pending status.